ktor-client/ktor-client-darwin/darwin/src/io/ktor/client/engine/darwin/certificates/CertificatePinner.kt

```diff
@@ -351,7 +343,7 @@ public data class CertificatePinner(
         val keyTypeRSA = CFBridgingRelease(kSecAttrKeyTypeRSA) as NSString
         val keyTypeECSECPrimeRandom = CFBridgingRelease(kSecAttrKeyTypeECSECPrimeRandom) as NSString
 
-        val size: Int = publicKeySize.intValue.toInt()
+        val size: Int = publicKeySize.intValue
```

Comment:
Please keep toInt or use convert()

Reply:
Fixed. It seems IDEA removed these calls on cleanup 🤔